cmd/flags/target.go

```diff
@@ -62,20 +61,19 @@ func toIntOrDefaultIfNull(value *int, defaultValue int) int {
 func (t *Target) getWarmupTargetOptions() warmup.TargetOptions {
 
 	return warmup.TargetOptions{
-		ReadinessProtocol:         t.ReadinessProtocol,
-		ReadinessHTTPPath:         t.ReadinessHTTPPath,
-		ReadinessGrpcMethod:       t.ReadinessGrpcMethod,
-		ReadinessPort:             t.ReadinessPort,
-		ReadinessTimeoutInSeconds: t.ReadinessTimeoutSeconds,
+		ReadinessProtocol:   t.ReadinessProtocol,
+		ReadinessHTTPPath:   t.ReadinessHTTPPath,
+		ReadinessGrpcMethod: t.ReadinessGrpcMethod,
+		ReadinessPort:       t.ReadinessPort,
 	}
 }
 
 func (t *Target) getReadinessHTTPClient() http.Client {
 	return http.NewClient(fmt.Sprintf("%s:%d", t.HTTPHost, t.ReadinessPort), t.Insecure)
 }
 
-func (t *Target) getReadinessGrpcClient() grpc.Client {
-	return grpc.NewClient(fmt.Sprintf("%s:%d", t.GrpcHost, t.ReadinessPort), t.Insecure, t.ReadinessTimeoutSeconds)
```

Comment:
The `ReadinessTimeoutSeconds` was removed so this is always 0. We now use the `-timeout-seconds`.